code/src/components/News/News.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import Headline from '../Shared/Headline/Headline'
+import Album from './Album/Album'
+import data from '../../data.json'
+
+class News extends Component {
+  constructor() {
+    super()
+    this.state = {
+      news: data.albums.items,
+      title: 'New albums & singles'
+    }
```

Comment:
Great that you were experimenting with state before we introduced it, but if I'm gonna be picky, these things needn't be in state since they never change.. State is a place for things which change - for everything else, you can use normal variables.